zerver/lib/integrations.py

```diff
@@ -830,3 +830,12 @@ def __init__(self, name: str, *args: Any, **kwargs: Any) -> None:
         )
     ],
 }
+
+
+def get_all_event_types(integration: Integration) -> Optional[List[str]]:
```

Comment:
`all_event_types_for_integration` would be a nicer name that's readable without having to check where this imported from.

Reply:
Agreed.